test/integration/css-modules/test/index.test.js

```diff
@@ -724,3 +724,64 @@ describe('Catch-all Route CSS Module Usage', () => {
     }
   )
 })
+
+describe('cssmodules-pure-no-check usage', () => {
+  ;(process.env.TURBOPACK_DEV ? describe.skip : describe)(
```

Comment:
We should use test manifest instead of using `describe.skip` with a env var flag

Reply:
Looks fine without the `describe.skip` 🙇🏼